sdk/purview/azure-analytics-purview-account/README.md

```diff
@@ -0,0 +1,105 @@
+# Azure Purview Account client library for Java
```

Comment:
`azure-analytics-purview-account.properies` file should be added as well in `src/main/resources` directory.

Reply:
Done